src/commands/classes/Contest.command.ts

```diff
@@ -260,108 +175,117 @@ export default class ContestCommand extends Command {
           while(reacCount.includes(max)){
             let winner = {};
             let x = reacCount.indexOf(max);
-            winner["emoji"] = reacEmoji[x];
-            winner["votes"] = reacCount[x];
+            winner['emoji'] = reacEmoji[x];
+            winner['votes'] = reacCount[x];
             let fileUUID = Object.keys(files)[x];
-            winner["file"] = Object.assign(new Object, files[fileUUID]);
-            winner["UUID"] = fileUUID;
+            winner['file'] = Object.assign(new Object, files[fileUUID]);
+            winner['UUID'] = fileUUID;
             delete files[fileUUID];
-            reacEmoji.splice(x,1);
-            reacCount.splice(x,1);
+            reacEmoji.splice(x, 1);
+            reacCount.splice(x, 1);
             winners.push(winner);
           }
 
         }
-        console.log("Successful");
 
-        console.log("Creating announcement message");
-        let places = ["1st Place", "2nd Place", "3rd Place"];
+        // Creating announcement message
+        let places = ['1st Place', '2nd Place', '3rd Place'];
         let mEmbed = new MessageEmbed();
-        let announcement = "\nThe contest has ended and our winners are: "
-        for(let x=0; x<winners.length; ++x){
-          let names = "";
+        let announcement = '\nThe contest has ended and our winners are: '
+        for(let x = 0; x < winners.length; ++x) {
+          let names = '';
           let place = places[x];
           names += `[${winners[x].file.name}](${winners[x].file.url})\n`;
-          while((x+1 < winners.length) && (winners[x].votes == winners[x+1].votes)){
+
+          while(x + 1 < winners.length && winners[x].votes == winners[x + 1].votes) {
             ++x;
             names += `[${winners[x].file.name}](${winners[x].file.url})\n`;
           }
           mEmbed.addField(`${place} | ${winners[x].votes} votes`, names);
       
         }
-        mEmbed.addField(":tada:CONGRATULATIONS:tada:", "\nLook forward to our next contest!");
-        console.log('Successful');
+        mEmbed.addField(':tada:CONGRATULATIONS:tada:', '\nLook forward to our next contest!');
         
-        for(let x=0; x<winners.length; ++x){
-
+        for(let x=0; x<winners.length; ++x) {
```

Comment:
Since this is a refactor and clean-up, put a space after all for, if, while statements. Also, there is some space issues here.